### DISCUSSION TITLE
Unexpected error: Operation not authorized. Please review your credentials and try again

### DISCUSSION THREAD

**Discussion by heshanthenura** [jreleaser/jreleaser#1607] [Q&A] [answered]
I provided username and password given by Sonatype https://central.sonatype.com/ and personal GitHub access token in `config.toml` file located in `.jreleaser`

What's the problem here? Should I make account on different platform?

![image](https://github.com/jreleaser/jreleaser/assets/75155192/5055c358-6ca1-480d-b425-c2a0f8a76370)
[trace.log](https://github.com/jreleaser/jreleaser/files/14423773/trace.log)

**Comment by aalmiray** [comment]
Can you log in into the Sonatype UI using the provided credentials?
Is your staging profile active in `https://oss.sonatype.org/` or `https://s01.oss.sonatype.org/`?

**Reply by heshanthenura** [reply]
oooo, should I provide credentials that I use to login or credential the I generate from the site. 

I provided the username and password generated from here. 👇
![image](https://github.com/jreleaser/jreleaser/assets/75155192/04fdf720-df20-456d-8146-c7f63fbe2645)

this is staging part
![image](https://github.com/jreleaser/jreleaser/assets/75155192/2aa577b7-e2a7-4064-9145-0c1ff9441cc7)

**Reply by aalmiray** [reply]
As far as I understand credentials created at [central.sonatype.com](http://central.sonatype.com/) are not yet compatible with OSSRH (which are the 2 other links I posted before). you need to register for OSSRH access following this guide https://central.sonatype.org/publish/publish-guide/

**Reply by heshanthenura** [reply]
So as you say I have to make an account here right? it's not free right? https://www.sonatype.com/products/sonatype-nexus-repository

**Reply by aalmiray** [reply]
No. Creating an account for OSSRH is always free.

**Reply by heshanthenura** [reply]
I followed the guide  https://central.sonatype.org/publish/publish-guide/ you mentioned, but I'm not sure which website to create an account on. Could you please provide the link to the relevant site? Thanks!

**Reply by aalmiray** [reply]
Did you create an account that lets you login into https://s01.oss.sonatype.org/? That's the OSSRH repository you must use.

**Reply by aalmiray** [reply]
Based on https://central.sonatype.org/register/legacy/#releasing-deployment-from-ossrh-to-the-central-repository-introduction the reason to request access to OSSRH instead of the Central Portal is that JReleaser does not yet support publishing to the Portal (this feature will be added in `1.12.0`).

**Reply by heshanthenura** [reply]
To create an account I need to use jira right? but they say its depreciated. Am I doing anything wrong ??
![image](https://github.com/jreleaser/jreleaser/assets/75155192/6aebee04-3b12-4701-9121-7d06170cd604)
![image](https://github.com/jreleaser/jreleaser/assets/75155192/e89adc65-069e-4723-8d62-04947aa2ff6c)

**Reply by aalmiray** [reply, marked as answer]
Follow the instructions at https://central.sonatype.org/register/legacy/ 
It says you must contact support via email.

**Reply by heshanthenura** [reply]
Thank you, I got valid credentials. Thanks for help
